models/intel/smartlab-action-recognition-0001/README.md

```diff
@@ -0,0 +1,73 @@
+# smartlab-action-recognition-0001 (composite)
+
+## Use Case and High-Level Description
+
+This is an smartlab action recogntion model that recognizes smartlab actions.
+The model uses smartlab dataset to predict actions.
+The model is classfier for 3 class actions.
+
+## Example of the input data
+<!-- ![](./assets/frame0001.jpg) -->
+
+## Example of the output
+
+`openvino`
+
+## Composite model specification
+
+| Metric                                         | Value              |
+| ---------------------------------------------- | ------------------ |
+| Accuracy on the DSI1867                        | TODO               |
```

Comment:
DSI1867 is internal dataset name, it should not be a part of public documentation
```suggestion
| Accuracy (internal test set)                        | TODO               |
```